### ISSUE TITLE
[Bug]: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]:

### ISSUE THREAD

**Issue by rmarkins-godaddy** [puppeteer/puppeteer#7986]
### Bug description

Steps to reproduce the problem:

1. Install latest `puppeteer-core`
2. attempt to path to `./lib/cjs/puppeteer/revisions`

### Puppeteer version

13.3.0

### Node.js version

14.19.0

### npm version

6.14.19

### What operating system are you seeing the problem on?

Linux, macOS

### Relevant log output

```shell
Full error


internal/modules/cjs/loader.js:456
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/cjs/puppeteer/revisions' is not defined by "exports" in /Users/rmarkins/dev/work/ghc/support/estimo/node_modules/puppeteer-core/package.json
    at new NodeError (internal/errors.js:322:7)
    at throwExportsNotFound (internal/modules/esm/resolve.js:332:9)
    at packageExportsResolve (internal/modules/esm/resolve.js:565:3)
    at resolveExports (internal/modules/cjs/loader.js:450:36)
    at Function.Module._findPath (internal/modules/cjs/loader.js:490:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/Users/rmarkins/dev/work/ghc/support/estimo/scripts/find-chrome-binary.js:6:33) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```
```

**Comment by rmarkins-godaddy** [comment]
You can see an example of this for `require` in https://github.com/mbalabash/estimo/blob/master/scripts/find-chrome-binary.js#L6

This affects those using `@size-limit/time` package. 

Subsequently is it possible just to add those to the `exports` path as it is useful information. Happy to open a PR w/ a fix if that's agreeable.

**Comment by OrKoN** [comment]
Ugh, sorry about that. We didn't realise that the internal modules were used directly. Let's see how we can fix it (happy to review a PR) but my first thought is: perhaps we should create an API for this use case?

**Comment by jan-molak** [comment]
@OrKoN an API or a dedicated export would be appreciated, thanks! We're seeing the same issue with WebdriverIO (and other frameworks that rely on it, like Serenity/JS), which relies on Puppeteer to run local tests. Here are the [`puppeteer-core` imports](https://github.com/webdriverio/webdriverio/search?l=TypeScript&q=puppeteer-core&type=code) WebdriverIO uses.

Perhaps they could be added to `exports`?

**Comment by OrKoN** [comment]
we are going to export everything by default like it was (https://github.com/puppeteer/puppeteer/pull/7991 and https://github.com/puppeteer/puppeteer/pull/7992 should solve the issue)

**Comment by OrKoN** [comment]
Change of plans: we are going to revert the change and re-land it later in a backwards-compatible way.

**Comment by rmarkins-godaddy** [comment]
closing as solved in 13.3.1. Thanks for the quick turnaround
